CrowsZA/functions/fn_spawnIEDClutter.sqf

```diff
@@ -0,0 +1,168 @@
+/*/////////////////////////////////////////////////
+Author: Landric
+			   
+File: fn_spawnIEDClutter.sqf
+Parameters: 
+	position		- ([int])
+	radius			- (int)
+	density			- (float) modifier for the amount of clutter to spawn
+	maxClutterSize	- (float) affects the size of clutter-type objects spawned
+	iedSize			- (string) valid values are, "small", "medium", "large"
+	iedType			- (string) valid values are, "urban", "dug-in", "random"
+	iedAmount		- (int) number if explosives to create
+
+Return: none
+
+Spawns item clutter (e.g. garbage piles) and hidden IEDs
+
+*///////////////////////////////////////////////
+params [["_pos",[0,0,0],[[]],3], ["_radius", 5, [0]], "_density", "_maxClutterSize", "_iedSize", "_iedType", "_iedAmount"];
+
+diag_log format ["CrowsZA-spawnIEDClutter: Spawning a cluster of %1 IEDs at position %2", _iedAmount, _pos];
```

Comment:
Not sure if needed? Would there be a benefit to put it in the log? Its not something that directly influence players, and as such don't need it to check for misuse/abuse. 

I'm just trying to be vary of how much we print to debug log, that might not be necessary, as it gets spammed quite enough already from all mods and arma itself ;-)

Reply:
Fair enough - I err on the side of verbose-ness when logging, but in this case it may not be needed